docs/specifications/deep-links.md

```diff
@@ -0,0 +1,190 @@
+
+The following parameters are **optional**:
+
+- `unit` - a specified denomination of the token to use, if applicable (default for IOTA is `Mi`, SMR is `SMR`)
+- `assetId` - the identifier of the asset to send, e.g. `4218` (IOTA), `4219` (SMR), or a native token ID (default is base token of the network, i.e. IOTA or SMR)
+- `metadata` - a string of text to embed as metadata in the transaction
+- `tag` - a string to tag the transaction for indexing purposes
+
+Example:
+
+[!button Click me!](firefly://wallet/sendForm?address=iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx&amount=10&unit=Gi)
+
+Source:
+
+```
+firefly://wallet/sendForm?address=iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx&amount=10&unit=Gi
+```
+
+#### Send Confirmation
+
+This operation brings the user to the send confirmation popup:
+
+:::image
+![](../static/send-confirmation-popup.png "Send confirmation popup")
+:::
+
+The deep link structure is as follows:
+
+```
+firefly://wallet/sendConfirmation?address=<address>&amount=<amount>[&unit=<unit>][&assetId=<assetId>][&metadata=<metadata>][&tag=<tag>][&giftStorageDeposit=<true|false>][&disableToggleGift=<true|false>][&disableChangeExpiration=<true|false>][&surplus=<surplus>]
+```
+
+The following parameters are **required**:
+
+- `address` - the recipient's address where the funds will be sent to
+  - **MUST** be a Bech32 address; considering support for other address types in the future
+- `amount` - the amount of tokens to send in the transaction
+  - **MAY** contain a decimal so long as it makes sense given the value of the `unit` param (see below)
+
+The following parameters are **optional**:
+
+- `unit` - a specified denomination of the token to use, if applicable (default for IOTA is `Mi`, SMR is `SMR`)
+- `assetId` - the identifier of the asset to send, e.g. `4218` (IOTA), `4219` (SMR), or a native token ID (default is base token of the network, i.e. IOTA or SMR)
+- `metadata` - a string of text to embed as metadata in the transaction
+- `tag` - a string to tag the transaction for indexing purposes
+- `giftStorageDeposit` - gifts the tokens used in funding the storage deposit for a transaction
+- `disableToggleGift` - prevents the user from being able to toggle the option to gift the storage deposit
+- `disableChangeExpiration` - prevents the user from being able to change the expiration time of the transaction
+- `surplus` - send additional amounts of the base token when transferring native tokens
+
+Example:
+
+[!button Click me!](firefly://wallet/sendForm?address=iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx&amount=10&unit=Gi&giftStorageDeposit=true&surplus=1&metadata=Take%20my%20money)
+
+Source:
+
+```
+firefly://wallet/sendConfirmation?address=iota1qrhacyfwlcnzkvzteumekfkrrwks98mpdm37cj4xx3drvmjvnep6xqgyzyx&amount=10&unit=Gi&giftStorageDeposit=true&disableToggleGift=true&surplus=1&metadata=Take%20my%20money
```

Comment:
you should put your own smr address here :rofl:

Reply:
Oh I'll do it :joy: